README.md

```diff
@@ -52,8 +52,8 @@ vars:
     company_pass_through_columns: [company_custom_field_1, company_custom_field_2]
     contact_pass_through_columns: [contact_custom_column]
```

Comment:
what's the difference between the two pairs of variables?

Reply:
i mean ie `company_pass_through_columns` vs `intercom__company_history_pass_through_columns`

Reply:
There is no difference. A local commit I had where I made the variables global was not pushed. This will be incorporated in the next push.